src/templates/caseStudyTemplate.js

```diff
@@ -2,136 +2,183 @@
 
 import React from "react"
 import _ from "lodash"
-import { graphql } from "gatsby"
-
+import { graphql, useStaticQuery } from "gatsby"
 import GeneralLayout from "./../components/layouts/GeneralLayout"
+import Markdown from "react-markdown"
 import ImageSlider from "./../components/organisms/ImageSlider"
 import RelatedByTitle from "./../components/RelatedByTitle"
 
-export default function Template({ data, location }) {
-  const { markdownRemark, allMarkdownRemark } = data
-  const { frontmatter, html } = markdownRemark
-  const { related_titles, specs, tags, images } = frontmatter
-  const { edges } = allMarkdownRemark
-
-  const specsList = _.map(specs, (spec, index) => (
-    <li className="study__item" key={index}>
-      {` `}
-      {spec}
-    </li>
-  ))
-
-  const tagsList = tags.map((tag, index) => (
-    <button className="tags" key={index}>
-      {tag}
-    </button>
-  ))
-
+export default function Template({ _, location }) {
+  const data = useStaticQuery(query)
+  console.log(data)
+  let caseStudy = {}
+  data.allStrapiCaseStudy.nodes.map((node, i) => {
+    if (node.Path == "/case-study/" + location.href.split("/")[4]) {
```

Comment:
I'd like to understand this approach better @Yodart. Is this trying to differentiate between old and new case studies?

Reply:
This is used to acquire the present case study from the query which comes with the others.